lib/generators/matestack_component/templates/matestack_component.js.erb

```diff
@@ -0,0 +1,12 @@
+MatestackUiCore.Vue.component('custom<% unless @namespace.nil? %>-<%= @namespace %><% end %>-<%= file_name %>', {
```

Comment:
The user should be informed, that he/she has to include this generated JS within the application.js

Reply:
Been there, done that. Also changed the Vue boilerplate to ES5, as you suggested